regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -32,74 +32,55 @@ var MainView = Backbone.View.extend({
-        childViewOptions.cfrTitle = this.cfrTitle;
-
-        appendixOrSupplement = this.isAppendixOrSupplement();
-        if (appendixOrSupplement) {
-            // so that we will know what the doc title format should be
-            childViewOptions.subContentType = appendixOrSupplement;
-        }
-
-        // find search query
-        if (this.contentType === 'search-results') {
-            childViewOptions.params = URI.parseQuery(window.location.search);
-            childViewOptions.query = childViewOptions.params.q;
-        }
-
-        if (this.contentType === 'landing-page') {
-            DrawerEvents.trigger('pane:change', 'table-of-contents');
-        }
-
-        // we don't want to ajax in data that the page loaded with
-        childViewOptions.render = false;
-
-        if (this.sectionId && this.modelmap[this.contentType]) {
-            // store the contents of our $el in the model so that we
-            // can re-render it later
-            this.modelmap[this.contentType].set(this.sectionId, this.$el.html());
-            childViewOptions.model = this.modelmap[this.contentType];
-        }
-
-        if (this.contentType && typeof this.viewmap[this.contentType] !== 'undefined') {
-            // create new child view
-            this.childView = new this.viewmap[this.contentType](childViewOptions);
-        }
-
-        this.sectionFooter = new SectionFooter({el: this.$el.find('.section-nav')});
+      this.dataTables = null;
+
+      if (Router.hasPushState) {
+        this.listenTo(MainEvents, 'search-results:open', this.createView);
+        this.listenTo(MainEvents, 'section:open', this.createView);
+        this.listenTo(MainEvents, 'diff:open', this.createView);
+        this.listenTo(MainEvents, 'breakaway:open', this.breakawayOpen);
+        this.listenTo(MainEvents, 'section:error', this.displayError);
+      }
+      this.listenTo(MainEvents, 'section:resize', this.applyTablePlugin);
+      this.listenTo(MainEvents, 'section:setHandlers', this.setHandlers);
+
+      this.$topSection = this.$el.find('section[data-page-type]');
+
+      // which page are we starting on?
+      this.contentType = this.$topSection.data('page-type');
+      // what version of the reg?
+      this.regVersion = Helpers.findVersion(Resources.versionElements);
+      // what section do we have open?
+      this.sectionId = this.$topSection.attr('id') ||
+        this.$topSection.find('section[id]').attr('id');
+      this.docId = $('#menu').data('doc-id');
+      this.cfrTitle = $('#menu').data('cfr-title-number');
+
+      if (this.sectionId && this.modelmap[this.contentType]) {
+        // store the contents of our $el in the model so that we
+        // can re-render it later
+        this.modelmap[this.contentType].set(this.sectionId, this.$el.html());
+        this.childModel = this.modelmap[this.contentType];
+      }
+
+      var options = {
+        el: '#' + this.sectionId,
```

Comment:
Interesting--because of the override on `setElement` in `ChildView` on `master`, setting `el` like this should have no effect. So it's not clear to me whether the intention of the current code is to set `el` to `#content-wrapper`, or to the id of the nested section--which typically aren't the same thing. Anyway, if we're willing to require that all child partials include a `#content-wrapper`, we should be able to stick with a fixed selector on `ChildView`.